docs/modules/maintain-cluster/pages/rolling-upgrades.adoc

```diff
@@ -107,6 +134,12 @@ include::clients:partial$rest-deprecation.adoc[]
 
 The cluster is now operating on the new codebase version: Both the codebase version and the cluster version have been upgraded.
 
+=== Restart Members on Previous Version
+
+If you experience any issues during the rolling upgrade, you can recover to the previous version by restarting the members.
```

Comment:
we are testing on between 5.2-5.4 source version to 5.5 target version upgrades. So should we add a note about this is supported only for LTS upgrades? In the future (post 5.5) though, it'll be supported for one-minor upgrades as well as I know.